client/my-sites/upgrades/checkout-thank-you/index.jsx

```diff
@@ -215,20 +234,24 @@ const CheckoutThankYou = React.createClass( {
 				<CheckoutThankYouHeader
 					isDataLoaded={ this.isDataLoaded() }
 					primaryPurchase={ primaryPurchase }
-					selectedSite={ this.props.selectedSite } />
+					selectedSite={ selectedSite } />
 
 				<CheckoutThankYouFeaturesHeader
 					isDataLoaded={ this.isDataLoaded() }
 					isGenericReceipt={ this.isGenericReceipt() }
-					purchases={ this.isGenericReceipt() ? false : getPurchases( this.props ) } />
+					purchases={ purchases } />
 
 				{ ComponentClass && (
 					<div className="checkout-thank-you__purchase-details-list">
 						<ComponentClass
+							domain={ domain }
 							purchases={ purchases }
 							registrarSupportUrl={ this.isGenericReceipt() ? null : primaryPurchase.registrarSupportUrl }
-							selectedSite={ selectedSite }
-							domain={ domain } />
+							selectedSite={ selectedSite } />
+
+						<FreeTrialNudge
```

Comment:
@breezyskies suggested that we display the nudge at [the bottom of the page](https://github.com/Automattic/wp-calypso/issues/3597#issuecomment-193360294) first.